ecosystem-modules/ren/renvm-bridge/src/lib.rs

```diff
@@ -37,6 +36,8 @@ pub trait Config: system::Config {
 
 decl_storage! {
 	trait Store for Module<T: Config> as Template {
+		/// The RenVM split public key
+		RenVmPublicKey get(fn ren_vm_public_key): Option<PublicKey>;
```

Comment:
```suggestion
		RenVmPublicKey get(fn ren_vm_public_key) config(): Option<PublicKey>;
```

No need extra genesis